cmd/blockstorage_snapshot_create.go

```diff
@@ -0,0 +1,96 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/exoscale/cli/pkg/globalstate"
+	"github.com/exoscale/cli/pkg/output"
+	v3 "github.com/exoscale/egoscale/v3"
+)
+
+type blockStorageSnapshotCreateCmd struct {
+	cliCommandSettings `cli-cmd:"-"`
+
+	_ bool `cli-cmd:"create"`
+
+	Volume string            `cli-arg:"#" cli-usage:"<volume NAME|ID>"`
+	Name   string            `cli-flag:"name" cli-usage:"block storage volume snapshot name"`
+	Labels map[string]string `cli-flag:"label" cli-usage:"block storage volume snapshot label (format: key=value)"`
```

Comment:
@arnaudgeiser is there a plan to support an `UpdateBlockStorageSnapshot` call? To update name, desc, labels?

Reply:
Yes, there is a plan to support both `UpdateBlockStorageVolume` and `UpdateBlockStorageSnapshot` for both the name and the labels.

Reply:
Thanks :)